reset.go

```diff
@@ -0,0 +1,59 @@
+package goose
+
+import (
+	"database/sql"
+	"log"
+	"sort"
+)
+
+// Reset rolls back all migrations
+func Reset(db *sql.DB, dir string) error {
+	migrations, err := CollectMigrations(dir, minVersion, maxVersion)
+	if err != nil {
+		return err
+	}
+	statuses, err := dbMigrationsStatus(db)
```

Comment:
Can you grab the current migration and perform `migration.Down()` and do the same thing for all `migration.Previous()` in a for loop? Just so we are consistent with the other commands.